test/one/profiler/test/Output.java

```diff
@@ -51,6 +60,19 @@ public double ratio(String regex) {
         return (double) matched / total;
     }
 
+    public String convert(ProfileOutputType fromOutputType, ProfileOutputType toOutputType, InputStream inputStream) {
+        try {
+            if (fromOutputType.equals(FLAMEGRAPH) && toOutputType.equals(COLLAPSED)) {
+                return new FlameToCollapsedConverter().convert(
+                        inputStream, new Arguments("-o", "collapsed"));
+            } else {
+                throw new RuntimeException("Conversion for the provided fromOutputType is not implemented!");
+            }
+        } catch (IOException e) {
+            return "";
```

Comment:
Exceptions should be propagated for the test to fail with the original exception.

Reply:
Rectified